### PR TITLE
Add accrual value

### PR DESCRIPTION
**Pull request by wegory**

### CODE REVIEW

File: contracts/LiquidityManager.sol [hunk 1]
```diff
@@ -141,7 +147,8 @@ library LiquidityManager {
                 pending: node.pending,
                 redemptions: node.redemptions.pending,
                 prev: node.prev,
-                next: node.next
+                next: node.next,
+                accrued: node.accrued
```
Comment (vsergeev) [inline contracts/LiquidityManager.sol:151]: We'll need to return accrualRate and accrualTimestamp as well to calculate this accurately offchain

File: contracts/LiquidityManager.sol [hunk 2]
```diff
@@ -106,6 +109,9 @@ library LiquidityManager {
         uint128 prev;
         uint128 next;
         Redemptions redemptions;
+        uint128 accrued;
```
Comment (vsergeev) [inline contracts/LiquidityManager.sol:112]: Could we organize this under an Accrual struct without impact on gas / size?

File: contracts/LiquidityManager.sol (hunk 1 above)
Comment (wegory) [reply contracts/LiquidityManager.sol]: Fixed and updated contract size screenshot in description above

**Comment by vsergeev** [comment]
Cherry-picked into `devel`.

**Pull request closed without merging**